tasks/install.yml

```diff
@@ -78,3 +78,11 @@
     - {src: "{{ role_path }}/templates/systemd/funkwhale-api.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_api_identifier }}.service"}
     - {src: "{{ role_path }}/templates/systemd/funkwhale-celery-beat.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_celery_beat_identifier }}.service"}
     - {src: "{{ role_path }}/templates/systemd/funkwhale-celery-worker.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_celery_worker_identifier }}.service"}
+    - {src: "{{ role_path }}/templates/systemd/funkwhale-watch-imports.service.j2", dest: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_watch_imports_identifier }}.service", when: funkwhale_watch_imports_enabled}
+
+- name: Ensure disabled funkwhale services are stopped and removed
+  when: not funkwhale_watch_imports_enabled
```

Comment:
I think you could have kept this `when` statement where it was (in the `with_items` item record) and added: `when: item.when | bool` here (untested, but would probably work).

This way, other services could be removed under other conditions.

Right now, this task appears to be generic (as if it could possibly remove other services), but the `when` condition is very specific to removing the imports watcher. I suppose it's OK to keep this task targeted at removing this watcher but I'd recommend updating its `name` to sound less generic in. And likely even removing the `with_items` loop and going for a simpler include, or even an inlined `block` if you'd rather skip the include.

By the way, the `remove_service.yml` task doesn't seem to exist in this PR.

Reply:
Thanks for explaining, I'm very new with ansible and learning on each commit. I can change the name to something more specific. Including `when: item.when | bool` skips the task when flipping `funkwhale_watch_imports_enabled`:
```
- name: Ensure disabled funkwhale watch imports service is stopped and removed
  when: item.when | bool
  ansible.builtin.include_tasks: "{{ role_path }}/tasks/remove_service.yml"
  with_items:
    - service: "{{ funkwhale_watch_imports_identifier }}.service"
      path: "{{ devture_systemd_docker_base_systemd_path }}/{{ funkwhale_watch_imports_identifier }}.service"
      when: not funkwhale_watch_imports_enabled
```
The current commit on the other hand works fine when flipping the value.

`remove_service.yml` already exists upstream and does not appear to require changes.